File: 27. Remove Element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        ind = 0
        for i in xrange(len(nums)):
            if nums[i] != val:
                nums[ind] = nums[i]
                ind += 1
        return ind

File: 27. Remove Element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    let ind = 0;
    for (let i = 0; i < nums.length; i++)
    {
        if (nums[i] != val)
        {
            nums[ind] = nums[i];
            ind += 1;
        }
    }
    return ind;
};
